edge.js

```diff
@@ -63,7 +63,7 @@ class EdgeWebDriverInstaller extends WebDriverInstallerBase {
    * @return {!Promise<string>}
    */
   async getBestDriverVersion(browserVersion) {
-    const idealMajorVersion = parseInt(browserVersion.split('.')[0], 10);
+    let idealMajorVersion = parseInt(browserVersion.split('.')[0], 10);
```

Comment:
very minor nit. This line can be moved closer to its usage before line 83.

Reply:
Done